cardano-node/app/cardano-node.hs

```diff
@@ -215,9 +229,9 @@ parsePort =
     )
 
 -- Optional flag for live view (with TUI graphics).
-parseViewMode :: Parser ViewMode
+parseViewMode :: Parser (Last ViewMode)
 parseViewMode =
-    flag SimpleView LiveView $ mconcat
+    flag (Last $ Just SimpleView) (Last $ Just LiveView) $ mconcat
```

Comment:
This looks kind of wordy, maybe try using existing functions from `CommonCLI`?

Reply:
@ksaric I will move `flagParser` to `CommonCLI` and import it here.

Reply:
@ksaric https://github.com/input-output-hk/cardano-node/pull/249#discussion_r335533540